jetty-server/src/main/config/etc/jetty-requestlog.xml

```diff
@@ -12,7 +12,7 @@
   <!-- =========================================================== -->
   <Set name="RequestLog">
     <New id="RequestLog" class="org.eclipse.jetty.server.AsyncNCSARequestLog">
-      <Set name="filename"><Property name="jetty.base" default="." /><Property>
+      <Set name="filename"><Property name="jetty.base" default="." />/<Property>
           <Name>jetty.requestlog.filePath</Name>
           <Deprecated>requestlog.filename</Deprecated>
           <Default><Property name="jetty.requestlog.dir" default="/logs"/>/yyyy_mm_dd.request.log</Default>
```

Comment:
wont this end up as `.//yyyy_mm_dd.request.log` if neither property is set?
If the / is added to line 15, then it should be removed from the default on 18.
Alternately encode a call to URIUtil.addPaths which does the handling for missing and/or double slashes... but that is probably overkill.